### ISSUE TITLE
The result I train the model from the github code is different with your paper, drop 1.2%.

### ISSUE THREAD

**Issue by rockywind** [xinzhuma/patchnet#18]
My environment is
python3.7
pytorch1.8
cuda11.1
The test result is
![image](https://user-images.githubusercontent.com/14838781/132832014-47646f24-ac3c-4eef-a067-9d2ab0672f16.png)

**Comment by xinzhuma** [comment]
it seems okay.  AP11 is not a stable metric for the monocular version. See [monodis, iccv'19] for the explanation. AP40 or stereo version is more reliable. Besides, our code is written and tested under pytorch1.1,  there may be some performance changes for different local environments.

**Comment by rockywind** [comment]
But I tested the model from Github. The accuracy is similar to your paper.
The result I tested on the pre-train model is below.
I found that the moderate AP40 was 16.60, higher than my model's result.
![image](https://user-images.githubusercontent.com/14838781/132849445-11e71a9f-7ee5-469f-91a4-e4a3b7286fbb.png)
